Configuration/Geometry/python/dict2026Geometry.py

```diff
@@ -878,7 +865,7 @@
     ("O2","T14","C6","M3","F3","I7") : "D44",
     ("O3","T15","C8","M3","F2","I10") : "D45",
     ("O3","T15","C9","M3","F2","I10") : "D46",
-    ("O3","T15","C10","M3","F3","I10") : "D47",
```

Comment:
@slomeo - this doesn't look right. I think you need to rebase your branch.

Reply:
@ianna : Yes, you are right. I restored the files dict2026Geometry.py and  GeometryConf.py present in CMSSW_11_0_X_2019-08-23-2300 (containing D47) and I added the new lines for D49.  I have created a new cmsExtended2026D49_cfi.py and all tests (geometry and overlaps) are fine.